media-tv/mythtv/mythtv-30.0_p20190808.ebuild

```diff
@@ -0,0 +1,369 @@
+
+MYTHTV_BRANCH="fixes/${P%.*}"
+
+DESCRIPTION="Homebrew PVR project"
+HOMEPAGE="https://www.mythtv.org"
+SRC_URI="https://github.com/MythTV/mythtv/archive/${BACKPORTS}.tar.gz -> ${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="0/${PV}"
+
+IUSE_INPUT_DEVICES="input_devices_joystick"
+IUSE_VIDEO_CAPTURE_DEVICES="v4l ivtv ieee1394 hdpvr hdhomerun vbox ceton"
+IUSE="alsa autostart bluray cdda cdr cec debug dvd egl exif fftw jack java lcd libass
+	lirc oss perl pulseaudio python raw systemd vaapi vdpau vpx +wrapper x264 x265 +xml xmltv +xvid
+	zeroconf ${IUSE_INPUT_DEVICES} ${IUSE_VIDEO_CAPTURE_DEVICES}"
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+	bluray? ( xml )
+	cdr? ( cdda )
+"
+
+COMMON="
+	acct-user/mythtv
+	dev-libs/glib:2
+	dev-libs/lzo
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtscript:5
+	dev-qt/qtsql:5[mysql]
+	dev-qt/qtwebkit:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	media-gfx/exiv2:=
+	media-libs/freetype:2
+	media-libs/libsamplerate
+	media-libs/taglib
+	>=media-sound/lame-3.93.1
+	sys-libs/zlib
+	virtual/mysql
+	virtual/opengl
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	x11-libs/libXv
+	x11-libs/libXxf86vm
+	x11-misc/wmctrl
+
+	alsa? ( >=media-libs/alsa-lib-1.0.24 )
+	bluray? (
+		media-libs/libbluray:=
+		dev-libs/libcdio:=
+		sys-fs/udisks:2
+	)
+	cec? ( dev-libs/libcec )
+	dvd? (
+		dev-libs/libcdio:=
+		sys-fs/udisks:2
+	)
+	egl? ( media-libs/mesa[egl] )
+	fftw? ( sci-libs/fftw:3.0=[threads] )
+	hdhomerun? ( media-libs/libhdhomerun )
+	ieee1394? (
+		>=media-libs/libiec61883-1.0.0
+		>=sys-libs/libavc1394-0.5.3
+		>=sys-libs/libraw1394-1.2.0
+	)
+	jack? ( media-sound/jack-audio-connection-kit )
+	lcd? ( app-misc/lcdproc )
+	libass? ( >=media-libs/libass-0.9.11:= )
+	lirc? ( app-misc/lirc )
+	perl? (
+		>=dev-perl/libwww-perl-5
+		dev-perl/DBD-mysql
+		dev-perl/HTTP-Message
+		dev-perl/IO-Socket-INET6
+		dev-perl/LWP-Protocol-https
+		dev-perl/Net-UPnP
+	)
+	pulseaudio? ( media-sound/pulseaudio )
+	python? (
+		${PYTHON_DEPS}
+		dev-python/lxml
+		dev-python/mysql-python
+		dev-python/urlgrabber
+		dev-python/future
+		dev-python/requests-cache
+	)
+	systemd? ( sys-apps/systemd:= )
+	vaapi? ( x11-libs/libva:=[opengl] )
+	vdpau? ( x11-libs/libvdpau )
+	vpx? ( <media-libs/libvpx-1.7.0:= )
+	x264? (	>=media-libs/x264-0.0.20111220:= )
+	x265? (	media-libs/x265 )
+	xml? ( >=dev-libs/libxml2-2.6.0 )
+	xvid? ( >=media-libs/xvid-1.1.0 )
+	zeroconf? (
+		dev-libs/openssl:0=
+		net-dns/avahi[mdnsresponder-compat]
+	)
+"
+RDEPEND="${COMMON}
+	!media-tv/mythtv-bindings
+	!x11-themes/mythtv-themes
+	media-fonts/corefonts
+	media-fonts/dejavu
+	media-fonts/liberation-fonts
+	x11-apps/xinit
+	autostart? (
+		net-dialup/mingetty
+		x11-apps/xset
+		x11-wm/evilwm
+	)
+	dvd? ( media-libs/libdvdcss )
+	xmltv? ( >=media-tv/xmltv-0.5.43 )
+"
+DEPEND="${COMMON}
+	dev-lang/yasm
+	virtual/pkgconfig
```

Comment:
This is defined in BDEPEND and can be removed from here.

Reply:
Done

Reply:
Still not done.